frontend/src/screens/subwallets/SubwalletList.tsx

```diff
@@ -0,0 +1,233 @@
+                  required
+                  autoComplete="off"
+                />
+              </div>
+              <LoadingButton loading={isLoading} type="submit">
+                Create Sub-wallet
+              </LoadingButton>
+            </form>
+          ) : (
+            <UpgradeCard
+              title="Need more Sub-wallets?"
+              description="Upgrade to Pro to unlock unlimited sub-wallets"
+            />
+          )}
+
+          {!!onboardedApps?.length && (
+            <>
+              <div className="grid grid-cols-1 lg:grid-cols-2 gap-4 items-stretch app-list">
+                {onboardedApps.map((app, index) => (
+                  <AppCard key={index} app={app} />
+                ))}
+              </div>
+            </>
+          )}
+        </>
+      )}
+
+      {showIntro && !onboardedApps?.length && (
+        <div>
+          <div className="flex flex-col gap-6 max-w-screen-md">
+            <div className="mb-2">
+              <img
+                src="/images/illustrations/sub-wallet-dark.svg"
+                className="w-72 hidden dark:block"
+              />
+              <img
+                src="/images/illustrations/sub-wallet-light.svg"
+                className="w-72 dark:hidden"
+              />
+            </div>
+            <div>
+              <div className="flex flex-row gap-3">
+                <Wallet2 className="w-6 h-6" />
+                <div className="font-medium">
+                  Sub-wallets are seperate wallets hosted by your Alby Hub
+                </div>
+              </div>
+              <div className="ml-9 text-muted-foreground text-sm">
+                Each sub-wallet has its own balance and can be used as a
+                separate wallet that can be connected to Alby Account or any
+                app.
+              </div>
+            </div>
+            <div>
+              <div className="flex flex-row gap-3">
+                <HandCoins className="w-6 h-6" />
+                <div className="font-medium">
+                  Sub-wallets depend on your Alby Hub spending balance and
+                  receive limit
+                </div>
+              </div>
+              <div className="ml-9 text-muted-foreground text-sm">
+                Sub-wallets are using your Hubs node liquidity. They can receive
+                funds as long as you have enough receive limit in your channels.
+              </div>
+            </div>
+            <div>
+              <div className="flex flex-row gap-3">
+                <TriangleAlert className="w-6 h-6" />
+                <div className="font-medium">
+                  Be wary of spending sub-wallets funds
+                </div>
+              </div>
+              <div className="ml-9 text-muted-foreground text-sm">
+                If your main balance runs low, funds from isolated wallets can
+                be spent by your main wallet balance. You'll receive alerts when
```

Comment:
is it true you'll receive alerts?

also it's not really true the funds will be spent, just that the sub-wallet balances _might not be spendable_ due to low funds inside channels

Reply:
Luckily you've really read this :pray: 

> is it true you'll receive alerts?
no :sweat_smile: 

I'll update the paragraph so it makes sense.